### PR TITLE
Ttva 27 price list templates

### PR DESCRIPTION
**Pull request by danjacob-anders**
https://andersinno.atlassian.net/jira/software/c/projects/TTVA/boards/11?modal=detail&selectedIssue=TTVA-27

### CODE REVIEW

File: respa_admin/templates/respa_admin/price_lists/_price_list_template.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+                    <li class="list-group-item">
+                        <div class="panel panel-default">
+                            <div class="panel-body">
+                                <div class="col-4">
+                                    <h4>{{ template.name }}</h4>
+                                </div>
+                                <div class="col-4">
+                                    <h6>
+                                        <span class='glyphicon glyphicon-plus plus' aria-hidden="true"></span>
+                                        <a href="{% url 'respa_admin:new-price-list-from-template' template.pk %}">{% trans 'Create price list' %}</a>
```
Comment (jopesy) [inline respa_admin/templates/respa_admin/price_lists/_price_list_template.html]: "Create price list" doesn't seem to have a Finnish translation. There's a translation for "Create a new price list", though.

**Comment by jopesy** [comment]
A couple of change suggestions for Respa admin to make it more clear the price list is based on a template:

image

File: respa_admin/tests/test_price_view.py [hunk 2]
```diff
@@ -1,11 +1,52 @@
...
+    return template
+
+
+@pytest.fixture
+def price_list_from_template(price_list_with_product, price_list_template):
+    return price_list_template.add_price_list(price_list_with_product)
+
+
+@pytest.mark.django_db
+def price_list_template_view(price_list_template, general_admin, rf):
```
Comment (jopesy) [inline respa_admin/tests/test_price_view.py]: `test_price_list_template_view`

File: respa_admin/templates/respa_admin/price_lists/_price_list_template.html (hunk 1 above)
Comment (danjacob-anders) [reply respa_admin/templates/respa_admin/price_lists/_price_list_template.html]: Yeah, it should just be "Create a new price list".

**Review by jopesy — APPROVED** [review]
Great work @danjacob-anders! 💯

**Pull request merged**